modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -57,82 +57,149 @@ podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal]
 ----
-podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
+oc-mirror --config=mirror-config.yaml file://mirror-archive
 ----
+. In your air-gapped environment, deploy the Helm chart to the `rhdh` namespace by running the `helm install` command. For example:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal,subs="attributes+"]
 ----
-podman pull registry.redhat.io/rhel9/postgresql-15:latest
+helm install rhdh ./mirror-archive/oc-mirror-workspace/src/charts/redhat-developer-hub-{product-version}.tgz --namespace rhdh --create-namespace
 ----
 
-. Push both images to the internal {ocp-short} image registry by running the following commands:
+.Verification
+. To verify that all resources are successfully deployed and running, check the status of your deployments and pods in the cluster.
+. To validate the route, ensure that the route matches the expected URL based on the router base of the cluster.
+.. If necessary, update the route to align with your expected configuration.
+
+== Installing {product} in a fully disconnected environment with the Helm chart
+
+You can use the Helm chart to install {product} by mirroring specified resources and transferring them to your air-gapped environment without any connection to the internet.
+
+.Prerequisites
+
+* You have installed {ocp-brand-name} {ocp-version-min} or later.
+* You have access to the `registry.redhat.io`.
+* You have access to the {ocp-brand-name} image registry for your cluster. For more information about exposing the image registry, see the {ocp-brand-name} documentation about https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry].
+* You have installed the {openshift-cli} on your workstation.
+* You have installed the `podman` command line tools on your workstation.
+* You have installed the oc-mirror {openshift-cli} plugin, for more information see https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-disconnected.html#installation-oc-mirror-installing-plugin_installing-mirroring-disconnected[Installing the oc-mirror OpenShift CLI plugin].
+* You have an account in https://developers.redhat.com/[{rhdeveloper-name}] portal.
+
+.Procedure
+
+. Log in to your {ocp-short} account using the {openshift-cli} by running the following command:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal]
 ----
-podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-version}
+oc login -u <user> -p <password> https://api.<hostname>:6443
 ----
+
+. Log in to the {ocp-short} image registry using the `podman` command line tool by running the following command:
```

Comment:
Same point related to the OCP image registry: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/868#discussion_r1932135022